.github/integration/tests/tests.sh

```diff
@@ -287,7 +290,7 @@ fi
 # Try downloading from private bucket
 printf "%s" "Attempting to download from S3 bucket with ACL=private..."
 errorMessage="reason: request failed with \`403 Forbidden\`, details: {Code:AllAccessDisabled"
```

Comment:
Put the message in single quotes to avoid having to escape the backticks:
```suggestion
errorMessage='reason: request failed with `403 Forbidden`, details: {Code:AllAccessDisabled'
```

Reply:
this made shellcheck complain so I left it as it was